platform-sdk/swirlds-base/src/main/java/com/swirlds/base/time/NanoClock.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.swirlds.merkledb;
+package com.swirlds.base.time;
```

Comment:
Since the class is now in base we should do some minimal changes:
- Null annotations
- JavaDoc
- Null checks

Reply:
Sorry for the miss, fixed!